great_expectations/datasource/fluent/sqlite_datasource.py

```diff
@@ -170,6 +172,7 @@ class SqliteDatasource(SQLDatasource):
     _TableAsset: Type[SqlTableAsset] = pydantic.PrivateAttr(SqliteTableAsset)
     _QueryAsset: Type[SqlQueryAsset] = pydantic.PrivateAttr(SqliteQueryAsset)
 
+    @public_api
```

Comment:
@billdirks If we are making a method public, does it not also mean that we have to add a properly-formatted docstring?  Thanks.

Reply:
Done. I was under the mistaken impression that we would inherit these but apparently not. 
```
In [1]: import great_expectations as gx

In [2]: print(gx.datasource.fluent.sqlite_datasource.SqliteDatasource.add_query_asset.__doc__)
--Public API--Adds a query asset to this datasource.

        Args:
            name: The name of this table asset.
            query: The SELECT query to selects the data to validate. It must begin with the "SELECT".
            order_by: A list of Sorters or Sorter strings.

        Returns:
            The query asset that is added to the datasource.
            The type of this object will match the necessary type for this datasource.
            eg, it could be a QueryAsset or a SqliteQueryAsset.
```

Reply:
@billdirks The content looks really good, but for some reason, I am not seeing this documentation in the actual `Python` module.  Please let me know if I am misunderstanding something here.  Thank you.